lib/redis.js

```diff
@@ -6,7 +6,7 @@ module.exports = CoreObject.extend({
   init: function(options, lib) {
     this._super();
     var redisOptions = {};
-    var redisLib     = lib;
+    var RedisLib     = lib;
```

Comment:
I capitalized this because you call the the constructor with `ioredis` instead of `createClient`. It made more sense to me to call `new RedisLib()` instead of `new redisLib()`.